lib/pr_checker.js

```diff
@@ -77,7 +77,7 @@ class PRChecker {
 
   checkReviews() {
     const {
-      pr, logger, reviewers: { rejected, approved }
+      pr, logger, reviewers: { rejected, approved, commentApproved }
```

Comment:
I don't think we need the extra export here, the information is already in `approved[i].review.source`, so we can derive data from the source of truth here.

Reply:
Well, I thought so at first, but that's what was breaking the tests, and would have need it to revise them completely.
Plus, IMHO, since I've separated things, I think it's semantically weird to mix properly approved reviews and commented reviews with LGTM. I mean, that's the whole point of getting these reviews out of the metadatas.